integrations/.github/pull_request_template.md

```diff
@@ -7,20 +7,20 @@
  Relates to issue #123 - 'Improve Salesforce Connector Efficiency'. If none, state 'None'. -->
 
 ## Type of Change
-- [ ] New Connector (Destination or Source Connector)
+- [x] New Connector (Destination or Source Connector)
```

Comment:
@praneeth-ais We are not suppose to update the template with the changes we need to update the description of the PR with these changes please revert this change